### PR TITLE
fix: 修改远程字段发送请求无效

### LINKED ISSUE #267: 🐛 [Bug]:  数据源中获取远程字段中的发送请求逻辑缺失，并且在发送过程中应该支持设置请求头，方便对于有权限校验等请求的支持
### Environment

Chrome120.0.6099.216（正式版本） (arm64)

### Version

v16.20.2

### Version

最新

### Link to minimal reproduction

- 新建数据源
- 点击获取远程字段
- 点击发送请求
   点击以后没有任何反应，查看源码，似乎缺少相关逻辑
image
image

### Step to reproduce

- 新建数据源
- 点击获取远程字段
- 点击发送请求
   点击以后没有任何反应，查看源码，似乎缺少相关逻辑
image
image

### What is expected

- 点击的时候能正常发送请求
- 能否支持对请求头的设置

### What is actually happening

_No response_

### Any additional comments (optional)

_No response_

### PR DESCRIPTION
**Pull request by ghost**
English | [简体中文](https://github.com/opentiny/tiny-engine/blob/develop/.github/PULL_REQUEST_TEMPLATE/PULL_REQUEST_TEMPLATE.zh-CN.md)

# PR

## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows our [Commit Message Guidelines](https://github.com/opentiny/tiny-engine/blob/develop/CONTRIBUTING.md)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)
- [ ] Built its own designer, fully self-validated

## PR Type

What kind of change does this PR introduce?

- [x] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

### What is the current behavior?

Issue Number: #267 

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: packages/plugins/datasource/src/DataSourceRemotePanel.vue [hunk 1]
```diff
@@ -181,6 +184,10 @@ export default {
         .load()
         .then((res) => {
           state.remoteData.result = Array.isArray(res?.data?.items) ? res.data.items[0] : res?.data || res
+
+          const remoteDataEditor = document.querySelector('#remote-data-editor')
```
Comment (chilingling) [inline packages/plugins/datasource/src/DataSourceRemotePanel.vue:195]: 这里能拿到 ref 吗？用 ref 会不会好一点？

Comment (ghost) [reply packages/plugins/datasource/src/DataSourceRemotePanel.vue:195]: 元素不在当前组件中。ref的确好一些，好溯源，但是也需要跨组件通信，是否有其他角度看ref更好

Comment (chilingling) [reply packages/plugins/datasource/src/DataSourceRemotePanel.vue:195]: 主要是怕后来的维护者不晓得有这么一层引用，后面一改 id 就失效了，或者你加个注释也行

Comment (ghost) [reply packages/plugins/datasource/src/DataSourceRemotePanel.vue:195]: ok，添加注释说明是请求结果代码编辑框。
同时添加了notify提示，请求成功会提示填充

**Review by hexqi — APPROVED** [review]

**Review by chilingling — APPROVED** [review]

**Pull request merged**